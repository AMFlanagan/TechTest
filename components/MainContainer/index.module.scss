.mainContainer {
  max-width: 1200px;
  margin: 0 auto;
  padding-top: 70px;
  padding: 70px 16px 0 16px;
}