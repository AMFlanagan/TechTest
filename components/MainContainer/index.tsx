import React from 'react'

import styles from './index.module.scss'

interface IProps {
  children?: JSX.Element | JSX.Element[]
}

const MainContainer: React.FC<IProps> = (props: IProps) => {
  return <div className={styles.mainContainer}>{props.children}</div>
}

MainContainer.displayName = 'MainContainer'

export default MainContainer
