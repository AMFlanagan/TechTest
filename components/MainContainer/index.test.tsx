import MainContainer from '.'
import { render } from '@testing-library/react'

describe('Event', () => {
  it('renders as expect', () => {
    const { getByTestId, asFragment } = render(
      <MainContainer>
        <div data-testid="child">child</div>
      </MainContainer>
    )

    expect(getByTestId('child')).toBeTruthy()
    expect(asFragment()).toMatchSnapshot()
  })
})
