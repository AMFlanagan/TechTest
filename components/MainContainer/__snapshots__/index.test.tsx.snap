// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event renders as expect 1`] = `
<DocumentFragment>
  <div
    class="mainContainer"
  >
    <div
      data-testid="child"
    >
      child
    </div>
  </div>
</DocumentFragment>
`;
